genconvert/input/my-t-my-d0-zawgyi.txt

```diff
@@ -186,18 +187,20 @@ $ukinzi ($consonant) > $1 \u1064 ;
 ## Phase 2: Further adjustments
 ::Null;
 
-# Handle consonant, subscripted consonant, medial ra
-($narrowconsonant) ($subscriptitem) ($highsignZ) $medialraZ > \u1083 $1 $2 $3 ;
-($wideconsonant) ($subscriptitem) ($highsignZ) $medialraZ > \u1084 $1 $2 ;
-($narrowconsonant) ($subscriptitem) $medialraZ > \u1081 $1 $2 ;
-($wideconsonant) ($subscriptitem) $medialraZ > \u1082 $1 $2 ;
+# Two medials
+\u103a \u103c > \u103c \u107d;
```

Comment:
Do you have a test case for this?

Reply:
Yes, case 114 and 115 are for that and also 101.